google/cloud/aiplatform/featurestore/entity_type.py

```diff
@@ -381,3 +386,836 @@ def delete_features(self, feature_ids: List[str], sync: bool = True,) -> None:
+            )
+        else:
+            raise ValueError(
+                f"The type of `feature_time` field should be: `str` or `datetime.datetime`, "
+                f"get {type(feature_time)} instead. "
+            )
+
+        if entity_id_field is not None:
+            import_feature_values_request.entity_id_field = entity_id_field
+
+        if disable_online_serving is not None:
+            import_feature_values_request.disable_online_serving = (
+                disable_online_serving
+            )
+
+        if worker_count is not None:
+            import_feature_values_request.worker_count = worker_count
+
+        return import_feature_values_request
+
+    @base.optional_sync(return_input_arg="self")
+    def _import_feature_values(
+        self,
+        import_feature_values_request: gca_featurestore_service.ImportFeatureValuesRequest,
+        request_metadata: Optional[Sequence[Tuple[str, str]]] = (),
+        sync: bool = True,
+    ) -> "EntityType":
+        """Imports Feature values into the Featurestore from a source storage.
+
+        Args:
+            import_feature_values_request (gca_featurestore_service.ImportFeatureValuesRequest):
+                Required. Request message for importing feature values.
+            request_metadata (Sequence[Tuple[str, str]]):
+                Optional. Strings which should be sent along with the request as metadata.
+            sync (bool):
+                Optional. Whether to execute this import synchronously. If False, this method
+                will be executed in concurrent Future and any downstream object will
+                be immediately returned and synced when the Future has completed.
+
+        Returns:
+            EntityType - The entityType resource object with imported feature values.
+        """
+        _LOGGER.log_action_start_against_resource(
+            "Importing", "feature values", self,
+        )
+
+        import_lro = self.api_client.import_feature_values(
+            request=import_feature_values_request, metadata=request_metadata,
+        )
+
+        _LOGGER.log_action_started_against_resource_with_lro(
+            "Import", "feature values", self.__class__, import_lro
+        )
+
+        import_lro.result()
+
+        _LOGGER.log_action_completed_against_resource(
+            "feature values", "imported", self
+        )
+
+        return self
+
+    def ingest_from_bq(
+        self,
+        feature_ids: List[str],
+        feature_time: Union[str, datetime.datetime],
+        bq_source_uri: str,
+        batch_create_feature_configs: Optional[
+            Dict[str, Dict[str, Union[bool, int, Dict[str, str], str]]]
+        ] = None,
+        feature_source_fields: Optional[Dict[str, str]] = None,
+        entity_id_field: Optional[str] = None,
+        disable_online_serving: Optional[bool] = None,
+        worker_count: Optional[int] = None,
+        request_metadata: Optional[Sequence[Tuple[str, str]]] = (),
```

Comment:
nit. Should the default be None ?

Reply:
We carried the default value `()` from GAPIC, https://github.com/googleapis/python-aiplatform/blob/main/google/cloud/aiplatform_v1/services/featurestore_service/client.py#L430, that allows us avoid change the `None` to `()` (requires an iterable data type) if not set across all SDK call.